pkg/oc/cli/admin/groups/sync/sync.go

```diff
@@ -421,19 +402,29 @@ func (o *SyncOptions) Run(cmd *cobra.Command, f kcmdutil.Factory) error {
 
 	// Now we run the Syncer and report any errors
 	openshiftGroups, syncErrors := syncer.Sync()
-	if o.Confirm {
-		return kerrs.NewAggregate(syncErrors)
-	}
+	if !o.Confirm {
+		list := &unstructured.UnstructuredList{
+			Object: map[string]interface{}{
+				"kind":       "List",
+				"apiVersion": "v1",
+				"metadata":   map[string]interface{}{},
+			},
+		}
+		for _, item := range openshiftGroups {
+			unstructuredItem, err := runtime.DefaultUnstructuredConverter.ToUnstructured(item)
+			if err != nil {
+				return err
+			}
+			list.Items = append(list.Items, unstructured.Unstructured{Object: unstructuredItem})
+		}
```

Comment:
@deads2k does this conversion to unstructured make sense?

Reply:
@deads2k we would be using this, I believe, to create a helper re-usable by other commands